src/components/Node/Node.tsx

```diff
@@ -33,86 +33,95 @@ const useStyles = makeStyles(
+        setExpanded,
+        interactive
+      },
+      ref
+    ) => {
+      const { label } = pathwayState;
+      const style = {
+        top: yCoordinate,
+        left: xCoordinate
+      };
 
-    let expandedNodeClass = '';
-    if (expanded) topLevelClasses.push(nodeStyles.expanded);
-    if (isActionable) {
-      topLevelClasses.push(styles.actionable);
-      expandedNodeClass = styles.childActionable;
-    } else {
-      expandedNodeClass = isOnPatientPath
-        ? styles.childOnPatientPath
-        : clsx(styles.childNotOnPatientPath, classes['child-not-on-patient-path']);
-    }
-    const isGuidance = isGuidanceState(pathwayState);
-    // TODO: how do we determine whether a node has been accepted or declined?
-    // for now:
-    // if it's a non-actionable guidance state on the path: accepted == has documentation
-    // if it's actionable, not guidance or not on the path: null
-    const wasActionTaken = isOnPatientPath && isGuidance && !isActionable;
-    const isAccepted = wasActionTaken ? documentation?.resourceType !== 'DocumentReference' : null;
+      const classes = useStyles();
+      const backgroundColorClass = isOnPatientPath
+        ? styles.onPatientPath
+        : clsx(styles.notOnPatientPath, classes['not-on-patient-path']);
+      const isActionable = isCurrentNode && !documentation;
+      const topLevelClasses = [styles.node, backgroundColorClass];
+      let expandedNodeClass = '';
+      if (expanded) topLevelClasses.push(nodeStyles.expanded);
+      if (isActionable) {
+        topLevelClasses.push(styles.actionable);
+        expandedNodeClass = styles.childActionable;
+      } else {
+        expandedNodeClass = isOnPatientPath
+          ? styles.childOnPatientPath
+          : clsx(styles.childNotOnPatientPath, classes['child-not-on-patient-path']);
+      }
+      const isGuidance = isGuidanceState(pathwayState);
+      // TODO: how do we determine whether a node has been accepted or declined?
+      // for now:
+      // if it's a non-actionable guidance state on the path: accepted == has documentation
+      // if it's actionable, not guidance or not on the path: null
+      const wasActionTaken = isOnPatientPath && isGuidance && !isActionable;
+      const isAccepted = wasActionTaken
+        ? documentation?.resourceType !== 'DocumentReference'
+        : null;
 
-    return (
-      <div className={topLevelClasses.join(' ')} style={style} ref={ref}>
-        <div
-          className={`${nodeStyles.nodeTitle} ${onClickHandler && nodeStyles.clickable}`}
-          onClick={onClickHandler}
-        >
-          <div className={nodeStyles.iconAndLabel}>
-            <NodeIcon pathwayState={pathwayState} isGuidance={isGuidance} />
-            {label}
+      return (
+        <div className={topLevelClasses.join(' ')} style={style} ref={ref}>
+          <div
+            className={`${nodeStyles.nodeTitle} ${interactive && nodeStyles.clickable}`}
+            onClick={(): void => {
```

Comment:
Looking more it looks like doing this might allow the removal of the `_key` prop too.  but I might just be missing something 🙂 .

Reply:
The onclickHandler causes the child to rerender.  The components passed through do not.  I realize now I could've memoized it but either way, this way prevents an unnecessary rerender of the child.

Reply:
This sounds like a case, as I think you're alluding to, where we could use `useCallback` on `onClickHandler`. ([The React docs mention it is useful for passing callbacks to children while preventing unnecessary rerenders](https://reactjs.org/docs/hooks-reference.html#usecallback)).

I would suggest trying that approach instead of passing in the other components for the reasons discussed.

Reply:
I actually did use useCallback to begin with since that seemed like the most correct solution but it, too, caused a bonus render.

Reply:
hmm, ok.  I just want to make sure we're not losing some modularity just to optimize and I'm not sure how costly this render is.  Any tool we use is going to have some overhead, but its a tradeoff.

Reply:
update: @KeeyanGhoreshi and I got this working with useCallback.